client/src/components/app/App.js

```diff
@@ -1,17 +1,73 @@
 import React from 'react';
-import '../../assets/css/App.css';
+
+import style from './App.module.css';
+
```

Comment:
remove extra new lines.

Reply:
It was intentionally put to distinguish group of same type of import